docs/about/license.md

```diff
@@ -85,8 +87,8 @@ additional licenses. Please review individual repositories for more information.
 | hsa-amd-aqlprofile | [AMD Software EULA](https://www.amd.com/en/legal/eula/amd-software-eula.html)
 
 Open sourced ROCm components are released via public GitHub
-repositories, packages on `https://github.com/ROCm` and other distribution channels.
-Proprietary products are only available with special permission from AMD. Currently, only
+repositories, packages on `https://repo.radeon.com` and other distribution channels.
```

Comment:
The URLs should be links, not just codetags.  Is this the MD syntax?

```suggestion
repositories, packages on [https://repo.radeon.com](https://repo.radeon.com) and other distribution channels.
```

Reply:
I updated it.